src/actuator/ActuatorExecutorsSimpleNgp.C

```diff
@@ -0,0 +1,110 @@
+// Copyright 2017 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS), National Renewable Energy Laboratory, University of Texas Austin,
+// Northwest Research Associates. Under the terms of Contract DE-NA0003525
+// with NTESS, the U.S. Government retains certain rights in this software.
+//
+// This software is released under the BSD 3-clause license. See LICENSE file
+// for more details.
+//
+#include <actuator/ActuatorExecutorsSimpleNgp.h>
+
+namespace sierra {
+namespace nalu {
+
+ActuatorLineSimpleNGP::ActuatorLineSimpleNGP(
+  const ActuatorMetaSimple& actMeta,
+  ActuatorBulkSimple& actBulk,
+  stk::mesh::BulkData& stkBulk)
+  : actMeta_(actMeta),
+    actBulk_(actBulk),
+    stkBulk_(stkBulk),
+    numActPoints_(actMeta_.numPointsTotal_)
+{
+}
+
+void
+ActuatorLineSimpleNGP::operator()()
+{
+  auto forceReduce = actBulk_.actuatorForce_.view_host();
+
+  actBulk_.zero_source_terms(stkBulk_);
+
+  update();
+
+  // set range policy to only operating over points owned by local fast turbine
+  auto fastRangePolicy = actBulk_.local_range_policy();
+
+
+  Kokkos::parallel_for(
+    "computeForcesActuatorNgpSimple", fastRangePolicy,
+    ActSimpleComputeForce(actBulk_, actMeta_));
+
+  actuator_utils::reduce_view_on_host(forceReduce);
+
+  const int localSizeCoarseSearch =
+    actBulk_.coarseSearchElemIds_.view_host().extent_int(0);
+
+  // === Always use SpreadActuatorForce() ===
+  // -- for both isotropic and anisotropic Guassians ---
+  Kokkos::parallel_for(
+    "spreadForcesActuatorNgpSimple", localSizeCoarseSearch,
+    SpreadActuatorForce(actBulk_, stkBulk_));
+  // -- Uncomment below to use ActSimpleSpreadForceWhProjection
```

Comment:
For the debug output, I'd prefer just a macro or passing a bool from the input file, like `debug_output_` used later

Reply:
Sure, I changed this to run based on an input flag.